website/index.html

```diff
@@ -101,14 +101,14 @@ <h1 id="deno"><a href="#deno">#</a>Deno</h1>
 
       <h2 id="install"><a href="#install">#</a>Install</h2>
 
-      <p id="install-shell"><a href="#install-shell">#</a>With Shell</p>
+      <p id="install-shell">With Shell</p>
       <pre>curl -fL https://deno.land/x/install/install.sh | sh</pre>
-      <p id="install-powershell"><a href="#install-powershell">#</a>With PowerShell</p>
```

Comment:
If you remove link, remove the `id` too, it isn't needed anymore

Reply:
Fixed.